src/themis/secure_comparator.c

```diff
@@ -38,7 +38,7 @@ static themis_status_t ed_sign(uint8_t pos, const uint8_t *scalar, uint8_t *sign
 {
 	uint8_t r[ED25519_GE_LENGTH];
 	ge_p3 R;
-	uint8_t k[64];
+	uint8_t k[64]; //temp buffer for group elements and hash
```

Comment:
It is easier this way to trace and compare the code - when someone wants to understand how it works, they have to create a map in their minds how specific variables are named in different implementations (this annoyed me in the past very much)